src/polycubed/src/server/Resources/Endpoint/Hateoas.h

```diff
@@ -0,0 +1,201 @@
+/*
+ * Copyright 2020 The Polycube Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
I would suggest to add a comment explaining the overall architecture for HATEOS, helping people to understand the flow of execution and where the key information is located.